docs/tabpy-tools.md

```diff
@@ -157,6 +158,114 @@ client.remove('WillItDefault')
 
 ```
 
+## Predeployed Functions
+
+To setup models download the latest version of TabPy and follow the [instructions](server-download.md)
+to install and start up your server. Once your server is running, navigate to the
+models directory and run setup.py.  If your TabPy server is running on the default
+config (default.conf), you do not need to specify a config file when launching the
+script. If your server is running using a custom config you can specify the config
+in the command line like so:
+
+```sh
+
+python setup.py custom.conf
```

Comment:
What about login and password parameters?

Reply:
users are prompted for their login/password if auth is turned on in their custom conf, they don't pass it as arguments to the setup file